### PR TITLE
update(v6-migration): clarify dataDirs usage with linking updates

### PR DESCRIPTION
**Pull request by convexshiba**
specifically clarify that `DataDirs` should generally be left empty.

### CODE REVIEW

**Comment by zakkarry** [comment]
I'm not against changing the language if there is something unclear, however "exclusively use torrentDir" when referring to dataDirs seems fairly clear. The result of exclusively using torrentDir means that dataDir would not be used, and thus be empty.

I think you're making a change that has a distinction without any difference. Did you not understand the term exclusive?

**Comment by convexshiba** [comment]
Since setting `dataDirs` when not needed could (would?) cause problem, it should be made clear as much as possible.

The change is mostly intended for users who are not familiar with the system (like me) and may not immediately get the idea that when it says "**exclusively**", it's implicitly meaning "between dataDirs and torrentDirs, only use torrentDir".

Now I know `dataDirs` is mentioned in relation to `torrentDir` in the above section.. But again for things that could introduce errors, it doesn't hurt to be clear every time so there's no 2 ways about it.

**Comment by zakkarry** [comment]
> Since setting `dataDirs` when not needed could (would?) cause problem, it should be made clear as much as possible.
> 
> The change is mostly intended for users who are not familiar with the system (like me) and may not immediately get the idea that when it says "**exclusively**", it's implicitly meaning "between dataDirs and torrentDirs, only use torrentDir".
> 
> Now I know `dataDirs` is mentioned in relation to `torrentDir` in the above section.. But again for things that could introduce errors, it doesn't hurt to be clear every time so there's no 2 ways about it.

Google defines exclusive as `restricted or limited to the person, group, or area concerned.` or `excluding or not admitting other things.` Can you explain how the term `exclusively` is not explicit in the way you say it needs to be? Familiarity with cross-seed wouldn't be required.

Were you unfamiliar with the term `exclusively`? Is this a terminology or vocabulary choice that caused the misunderstanding? 

Where is the disconnect? I'm trying to understand why and what specifically failed to click when you were reading it.

**Comment by convexshiba** [comment]
I'm not saying the original was inaccurate.

I'm saying it can be made more explicit thus harder to make mistakes.

If you dislike the redundancy that's fine.

**Comment by zakkarry** [comment]
I'm just trying to understand what the difference is and what caused the confusion given what I understand to be a very specific definition of the word, and you're choice to change it for further clarity.

If you prefer to close the PR instead of discuss how our language choices can improve overall throughout our maintaining of the documentation, that's entirely your choice.

**Comment by convexshiba** [comment]
So my struggle with this sentence:

> **Because of the above updates to linking, we recommend that you **exclusively**
use `torrentDir`, except for

is that it did not directly tell me, in what context, among what options, I must "exclusively" use `torrentDir`.

In your comment's definition for exclusive, it says `excluding or not admitting other things`, the issues I think here is it didn't, directly, specify what are the ""other things".

Yes, in my case with my current knowledge, and in your case as dev, and when taking the above section reading as a whole, one can reasonably infer that here it actually means "among `torrentDir` and `dataDir`, you must exclusively use torrentDir".

But back when I was first configuring this, the question in my head is that "there're like 20 other params, what does it mean that I should exclusively use `torrentDir`, maybe it's saying that I must not use dataDir since it's mentioned in the above section, but I don't know for sure".

My edit's intention is that, if the case is as clear as `use torrentDir only and do not use dataDir`, let's directly and clearly state that.

**Comment by zakkarry** [comment]
I definitely understand the distinction you're attempting to make here now, and that does seem valid.

**Comment by zakkarry** [comment]
I'm in the middle of something right now, but I can look into reviewing this further at some point this weekend, possibly tomorrow morning—but I cannot promise a specific timeline.

If you'd like to address any adjustments in the meantime, feel free.

Thank you for explaining.

File: docs/v6-migration.md [hunk 1]
```diff
@@ -175,14 +175,14 @@ below).
...
 
 Because of the above updates to linking, we recommend that you **exclusively**
-use `torrentDir`, except for two specific use-cases below. Torrent based
-matching is more robust, and prevents some performance issues with `dataDirs`
-wherein `cross-seed` has to frequently scan your `dataDirs` and all of their
-children for changes.
+use `torrentDir` and leave `dataDirs` empty, except for two specific use-cases 
+below. Torrent based matching is more robust, and prevents some performance issues 
+with `dataDirs` wherein `cross-seed` has to frequently scan your `dataDirs` and 
+all of their children for changes.
```
Comment (zakkarry) [inline docs/v6-migration.md:181]:
```suggestion
Due to the linking updates mentioned, we recommend using **ONLY** `torrentDir` and
leaving `dataDirs` empty, except in the two cases outlined below. Torrent based
matching is more robust, and prevents some performance issues with `dataDirs`
wherein `cross-seed` has to frequently scan your `dataDirs` and all of their
children for changes.
```
Does this keep it as clear? Minor language changes.

**Review by zakkarry — CHANGES_REQUESTED** [review]

**Review by convexshiba — COMMENTED** [review]
adopt suggestions.

**Review by zakkarry — APPROVED** [review]

**Pull request merged**